lightly_studio/tests/resolvers/video/video_frame_resolver/test_get_all_by_dataset_id.py

```diff
@@ -106,6 +111,85 @@ def test_get_all_by_dataset_id(test_db: Session) -> None:
     assert result.samples[3].parent_sample_id == sample_video_2_id
 
 
+def test_get_all_by_dataset_id__with_frame_number_filter(test_db: Session) -> None:
+    dataset = create_dataset(session=test_db, sample_type=SampleType.VIDEO)
+    dataset_id = dataset.dataset_id
+
+    # Create videos
+    video_frame_data = create_video_with_frames(
+        session=test_db,
+        dataset_id=dataset_id,
+        video=VideoStub(path="/path/to/sample1.mp4"),
+    )
+
+    (frame_sample_id_1, frame_sample_id_2) = video_frame_data.frame_sample_ids[0:2]
+    video_frame_dataset_id = video_frame_data.video_frames_dataset_id
+
+    min_frame_number, max_frame_number = (0, 1)
+
+    samples = video_frame_resolver.get_all_by_dataset_id(
+        session=test_db,
+        dataset_id=video_frame_dataset_id,
+        video_frame_filter=VideoFrameFilter(
+            frame_number=FilterDimensions(min=min_frame_number, max=max_frame_number),
+        ),
+    ).samples
+
+    assert len(samples) == 2
+    assert samples[0].sample_id == frame_sample_id_1
+    assert samples[1].sample_id == frame_sample_id_2
```

Comment:
An important information is missing in the test - we don't know how many frames does the video have, so we don't know if the filter does anything.

Suggestion: Change the `VideoStub` to create a video with 5 frames and select only frames 2-3 from it.